pages/common/zpool.md

```diff
@@ -4,30 +4,28 @@
 
 - Show the configuration and status of all ZFS zpools:
 
-`zpool status [{{poolname}}]`
+`zpool status`
 
 - Check a ZFS pool for errors (verifies the checksum of EVERY block). Very CPU and disk intensive:
 
-`zpool scrub {{poolname}}`
+`zpool scrub {{pool_name}}`
 
 - List zpools available for import:
 
 `zpool import`
 
-- Import a zpool, optionally specifying a new name:
+- Import a zpool (optionally renaming it):
 
-`zpool import {{poolname}}`
-`zpool import {{poolname}} {{newpoolname}}`
+`zpool import {{pool_name}} {{[new_pool_name]}}`
```

Comment:
I feel this is kinda cheating. There shouldn't be two commands in this example anyway, so this isn't your fault @notpeter, but I'd rather we come to an agreement on this. I don't think we should include examples with optional parameters, as that could open a can of worms, but I'm open to reconsider.